packages/amplication-client/src/VersionControl/PendingChangeContent.tsx

```diff
@@ -23,7 +23,7 @@ const PendingChangeContent = ({
 }: Props) => {
   const { currentWorkspace, currentProject } = useContext(AppContext);
 
-  const url = `/${currentWorkspace?.id}/${currentProject?.id}/${change.resource.id}/${relativeUrl}`;
+  const url = `/${currentWorkspace?.id}/${currentProject?.id}/${relativeUrl}`;
```

Comment:
In the issue, the requirement is that clicking the Project Settings in the 'Pending Changes' panel should open [https://app.amplication.com/[workspace](https://app.amplication.com/%5Bworkspace) id]/[project id]/settings/update